plugins/by-name/efmls-configs/default.nix

```diff
@@ -44,10 +44,13 @@ in
       # Filter package defaults that are not compatible with the current platform
       (lib.attrsets.mapAttrs (
         _: opt:
-        if lib.meta.availableOn pkgs.stdenv.hostPlatform opt.default then
-          opt
-        else
-          builtins.removeAttrs opt [ "defaultText" ] // { default = null; }
+        opt
+        // {
+          default = if lib.meta.availableOn pkgs.stdenv.hostPlatform opt.default then opt.default else null;
+          defaultText = lib.literalMD ''
+            `${opt.defaultText.text}` if available on the current system, otherwise null
+          '';
+        }
```

Comment:
Do we _really_ need to nullify the default on unsupported platforms? It'd be much simpler to just set the default (unconditionally) and let users nullify it themselves if need be.

cc @traxys

Reply:
I think it was @GaetanLepage that added that? I would not be opposed to changing the behaviour.
The tests may rely on it though